pages/contact.html

```diff
@@ -12,11 +13,11 @@
 </head>
 
 <body>
-
+    
     <div class="ui container">    
     <img class="ui centered medium image" src="../assets/images/logo.jpg">
 
-        <div class="stackable ui tabular menu">
+        <div class="inverted blue stackable ui tabular menu">
```

Comment:
I don't think this change should be in this pr as this one relates to the navbar.

Reply:
@laurafitzgerald I changed it so it matched the footer on the page to see how it would look like.